Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -210,9 +211,10 @@ def self.check_http_content(url, user_agents: [:default])
     "The URL #{url} could use HTTPS rather than HTTP"
   end
 
-  def self.http_content_headers_and_checksum(url, user_agent: :default)
+  def self.http_content_headers_and_checksum(url, hash_needed = false, user_agent: :default)
+    header_strategy = hash_needed ? "--include" : "-I"
```

Comment:
You can also use `--range` but, again, I don't think it was reliable as some hosts didn't support it.